pkg/api/roles.go

```diff
@@ -0,0 +1,40 @@
+package api
```

Comment:
I assume rest of the roles need to be migrated here from accesscontrol package in a different PR?

Reply:
Yes they will :slightly_smiling_face: